packages/fx-core/src/plugins/resource/frontend/constants.ts

```diff
@@ -97,6 +97,14 @@ export class DependentPluginInfo {
   static readonly LocalTabEndpoint = "localTabEndpoint";
 }
 
+export class ArmOutput {
+  static readonly simpleAuthEndpoint = "simpleAuth_endpoint";
```

Comment:
We use PascalCase for constants in frontend plugin.

Reply:
this key is defined in arm output. I follow it.
@blackchoey  in the future, can we provide a whole arm output key names in common?

Reply:
I'm not sure whether @hund030 means the variable name `simpleAuthEndpoint` or the value `simpleAuth_endpoint`. If it's the value, do you think it's appropriate for us update the name in next week? So we can reduce some integration work.

Reply:
I mean the variable name `simpleAuthEndpoint`.

Reply:
OK. I fixed the names. Thanks.